src/js/configUtils.js

```diff
@@ -0,0 +1,95 @@
+/*
+Copyright 2015 OCAD University
+
+Licensed under the New BSD license. You may not use this file except in
+compliance with this License.
+
+You may obtain a copy of the License at
+https://raw.githubusercontent.com/GPII/first-discovery-server/master/LICENSE.txt
+*/
+
+"use strict";
+
+var fluid = require("infusion");
+var ajv = require("ajv");
+var gpii = fluid.registerNamespace("gpii");
+
+/**
+ * Exposes the environment variables available to the node process, to a
+ * form that is accessible via the IoC system.
+ */
+fluid.defaults("gpii.firstDiscovery.environment", {
```

Comment:
All of this stuff can just be put in plain gpii.* rather than gpii.firstDiscovery.* since it consists of general utilities that will be helpful across the project